lpa-submissions-e2e-tests/cypress/integration/pageFooter/pageFooter.step.js

```diff
@@ -0,0 +1,18 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+import { clickFooterLink } from '../../support/common/clickFooterLink';
+
+Given('LPA planning officer accesses the LPA Questionnaire', () => {
```

Comment:
This might be worth being a common step - if it isn't already?

Reply:
we can do it separately if there is a need